backend/routes/index.js

```diff
@@ -2,23 +2,26 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const Message = require('./messages')
 
+const pgClient = require("./connection")
+
 const router = express.Router();
 router.use(bodyParser.json());
 
 // Handles GET requests to /messages
-router.get('/messages', (req, res) => {
+router.get('/messages', async (req, res) => {
     console.log(`received request: ${req.method} ${req.url}`)
 
     // Query for messages in descending order
     try {
         // TODO: CloudSQL query
-        const msg = {
-            name: "Crisco",
-            body: "Northern pikas are most active and mostly feed soon after dawn and as dusk approaches.",
-            stickerUrl: "https://i.pinimg.com/originals/63/b3/49/63b349f74f7f2e498e1ca74c66b829fa.jpg",
-            timestamp: "11-19-2019"
-        };
-        res.status(200).json([msg]);
+        // const msg = {
+        //     name: "Crisco",
+        //     body: "Northern pikas are most active and mostly feed soon after dawn and as dusk approaches.",
+        //     stickerUrl: "https://i.pinimg.com/originals/63/b3/49/63b349f74f7f2e498e1ca74c66b829fa.jpg",
+        //     timestamp: "11-19-2019"
+        // };
+        const msg = await Message.retrieve(pgClient);
+        res.status(200).json(msg);
     } catch (error) {
         res.status(500).json(error);
```

Comment:
this currently does not throw an error if not connected to cloudsql

Reply:
Fixed this. See new unit test.